pegasus/sites.v3/code.org/public/about/careers/engineering.haml

```diff
@@ -0,0 +1,82 @@
+---
+title: Join Our Engineering Team!
+theme: responsive_full_width
+---
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/engineering-styles.css", rel: "stylesheet"}
+
+%section.hero-banner-basic.hero-banner-basic--engineering
+  .wrapper
+    .flex-container
+      .text-wrapper.col-60.flex-container.align-items-center
+        .hero-banner-basic__content
+          %h1{style: "color: white"}
+            =hoc_s(:engineering_hero_heading)
+          %p{style: "color: white"}
+            =hoc_s(:engineering_hero_desc)
+          %a{href: "/about/careers#roles", class: "link-button", target: "_blank", rel: "noopener noreferrer", aria:{label: hoc_s("call_to_action_open_positions")}}
```

Comment:
Not blocking, but you can use the new `.white` class on this button link here and remove the corresponding styles on lines 16-20 in the css: https://github.com/code-dot-org/code-dot-org/blob/d2de90f294fe95b6df74d3fc7d3402fa1864c082/pegasus/sites.v3/code.org/public/css/page/engineering-styles.scss#L16-L20